homeassistant/bootstrap.py

```diff
@@ -7,8 +7,10 @@
 from collections import defaultdict
 from threading import RLock
 
+# pylint: disable=ungrouped-imports
 from types import ModuleType
 from typing import Any, Optional, Dict
+from homeassistant.helpers.typing import HomeAssistantType
```

Comment:
I think we should just import this from `Core` and make sure that Core is actually core-worthy by not having any imports from `homeassistant.helpers.*`.